input/fsh/examples/at-aps-example-allergyintolerance-01.fsh

```diff
@@ -4,11 +4,9 @@ Title: "AtApsAllergyIntolerance-Beispiel"
 Description: "Überwundene Allergie"
 Usage: #example
 * id = "at-aps-example-allergyintolerance-01"
-* extension.url = "http://hl7.org/fhir/StructureDefinition/allergyintolerance-abatement"
-* extension.valueDateTime = "2020"
 * clinicalStatus = $cs-allergyintolerance-clinical#resolved
 * verificationStatus = $cs-allergyintolerance-verification#confirmed
 * code = $cs-sct#256303006 "Ambrosiapollen"
-* patient = Reference(Patient/at-aps-example-patient-01) "Maria Musterfrau"
+* patient = Reference(Patient/HL7ATCorePatientExample01) "Maria Mustermann"
```

Comment:
```suggestion
* patient = Reference(Patient/HL7ATCorePatientExample01) "Max Mustermann"
```
https://fhir.hl7.at/HL7-AT-FHIR-Core-R5/Patient-HL7ATCorePatientExample01.html is named "Max Mustermann". Please, check other occurrences as well.